drivers/syslog/Kconfig

```diff
@@ -102,15 +102,59 @@ config SYSLOG_TIMESTAMP
 	---help---
 		Prepend timestamp to syslog message.
 
+choice
+	prompt "Timestamp format"
+	depends on SYSLOG_TIMESTAMP
+
+config SYSLOG_TIMESTAMP_SYSTIME
+	bool "System time"
+	default y
+	---help---
+		Use the system timer for timestamp.
+
 config SYSLOG_TIMESTAMP_REALTIME
-	bool "Use wall-clock for syslog timestamp"
+	bool "Wall-clock time"
 	default n
-	depends on SYSLOG_TIMESTAMP
 	---help---
 		Use wall-clock (CLOCK_REALTIME) for timestamp.  By default,
 		CLOCK_MONOTONIC, if enabled, will be used or the system timer
 		is not.
 
+config SYSLOG_TIMESTAMP_FORMATED
+	bool "Formatted wall-clock time"
```

Comment:
Remove wall-clock since the format apply to real time, mononic time and system time.